contributors.md

```diff
@@ -0,0 +1,27 @@
+# Contributing to Docs Authoring Pack 
+
+Interested in making a code contribution and not sure where to begin? Check out our help wanted issues. If you want to make an enhancement, create an issue with the proposed change for review. 
```

Comment:
Hey @adunndevster , do you think it would be helpful to add a filtered link to the help want issues i.e. [help wanted issues](https://github.com/microsoft/vscode-docs-authoring/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)

Reply:
Yes!